temperature_control/janis_ctrl.py

```diff
@@ -492,38 +528,147 @@ def temperature_controller(self, tsidx, tidx, Tidx, t, Tset, fid, out):
         out['PID']      = [P, I, D]
         out[flo_key]    = flow
         fid.write(f'{tsout}, {tout}, {Tout}, {Iout}, {P}, {I}, {D}, {flow}\n')
+
+
+    def estimate_init_adaptive_averages(
```

Comment:
Where is IFBW taken into account here?

Reply:
The IFBW is taken into account in the user scripts supplying this method. I need to add those scripts to the repo at some point.